sympy/functions/combinatorial/factorials.py

```diff
@@ -172,6 +172,13 @@ def _eval_is_positive(self):
         if self.args[0].is_integer and self.args[0].is_nonnegative:
             return True
 
+    def _eval_is_composite(self):
+        x = self.args[0]
+        if x.is_integer:
+            return (x - 3).is_nonnegative
+        elif x.is_nonpositive or x.is_noninteger:
+            return False
```

Comment:
On Mon, Jan 05, 2015 at 12:14:54PM -0800, AMiT Kumar wrote:
>    I didn't considered negatives.

Actually, I'm not just about negatives.